shardingsphere-db-protocol/shardingsphere-db-protocol-mysql/src/test/java/org/apache/shardingsphere/db/protocol/mysql/constant/MySQLServerInfoTest.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.shardingsphere.db.protocol.mysql.constant;
+
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class MySQLServerInfoTest {
```

Comment:
Please add final if the class is not for extend

Reply:
I see , done